src/components/Api.js

```diff
@@ -0,0 +1,169 @@
+import { renderLoading } from "../utils/utils.js";
+
+import { api } from "../pages/index.js";
+import {
+  profileEditForm,
+  cardsEditForm,
+  propicForm,
+} from "../utils/constants.js";
```

Comment:
Please, delete the whole import from `Api.js` because it's responsible only for defining methods and checking responses 

The promise chain (then, catch, finally) should be only where you **call** these methods in `index.js`


From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.